umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -72,5 +73,11 @@
         update-format (or update-format (:format concept))
         umm (spec-core/parse-metadata
              context concept-type format metadata {:sanitize? (= :umm-json (mt/format-key update-format))})
-        umm (apply-update update-type umm update-field update-value find-value)]
+        metadata-dates (:MetadataDates umm)
```

Comment:
I'm not sure yet that we want to update the metadata and am talking to Kathy about it today. Please hold off on merging this change.